Sources/Common/DataModel/PiecewiseFunction/index.js

```diff
@@ -167,8 +167,8 @@ function vtkPiecewiseFunction(publicAPI, model) {
     const oldX = model.nodes[index].x;
     model.nodes[index].x = val[0];
     model.nodes[index].y = val[1];
-    model.nodes[index].Midpoint = val[2];
-    model.nodes[index].Sharpness = val[3];
+    model.nodes[index].midpoint = val[2];
+    model.nodes[index].sharpness = val[3];
```

Comment:
Are you sure that nowhere else Midpoint/Sharpness is used? It should have been lower case but since it is a breaking change. We need to make sure vtk.js is consistent internally...

Reply:
I grepped the source code and found no usage of `Midpoint` and `Sharpness` in capitalized form. For that matter, all tests pass as well.